client/my-sites/exporter/post-type-options.jsx

```diff
@@ -12,18 +12,22 @@ import FormRadio from 'components/forms/form-radio';
 import Select from './select';
 import Label from 'components/forms/form-label';
 
-import { setPostType } from 'state/site-settings/exporter/actions';
+import { setPostType, setPostTypeFilters } from 'state/site-settings/exporter/actions';
 import {
 	getPostTypeOptions,
+	getPostTypeValues,
 	getSelectedPostType,
 } from 'state/site-settings/exporter/selectors';
 
 const mapStateToProps = ( state, ownProps ) => {
 	const siteId = state.ui.selectedSiteId;
 	const options = getPostTypeOptions( state, siteId, ownProps.postType );
+	const values = getPostTypeValues( state, siteId, ownProps.postType );
```

Comment:
This could probably be named a bit clearer. Are these `values` the names of the post types? How about `postTypes` instead. If not, maybe something that gives more indication what they contain. Could also say the same about `options`, maybe something like `postTypeOptions`

This is an API here of sorts, because other people and other code might consume the interface, in which case I think having more specificity is valuable. Were it all contained inside of this module, I wouldn't care as much, but since it's a prop and we will hopefully do things like write unit tests for it eventually, that is where I am more sensitive to naming.